src/model/ThermalZone.cpp

```diff
@@ -2649,6 +2651,81 @@ namespace model {
       return zoneProp;
     }
 
+    boost::optional<double> ThermalZone_Impl::getAutosizedValueFromZoneSizes(const std::string& columnName, const std::string& loadType) const {
+      // Check that the model has a sql file
+      if (!model().sqlFile()) {
+        LOG(Warn,
+            "This model has no sql file, cannot retrieve the ZoneSizes autosized value '" << columnName << "' with loadType '" << loadType << "'.");
+        return boost::none;
+      }
+
+      if ((loadType != "Cooling") && (loadType != "Heating")) {
+        LOG(Warn, "Cannot retrieve the ZoneSizes autosized value '" << columnName << ", Invalid loadType, must be one of ['Cooling', 'Heating']");
+        return boost::none;
+      }
+
+      // Get the object name and transform to the way it is recorded
+      // in the sql file
+      std::string sqlName = name().get();
+      boost::to_upper(sqlName);
+
+      const std::string directQuery = fmt::format(
+        R"sql(
+SELECT {} FROM ZoneSizes
+  WHERE ZoneName = ?
+    AND LoadType = ?;
+      )sql",
+        columnName);
+      boost::optional<double> val = model().sqlFile().get().execAndReturnFirstDouble(directQuery,
+                                                                                     // bindArgs
+                                                                                     sqlName, loadType);
+      if (!val) {
+        LOG(Debug, fmt::format(R"sql(The direct query failed:
+SELECT {} FROM ZoneSizes
+  WHERE ZoneName = '{}'
+    AND LoadType = '{}';)sql",
+                               columnName, sqlName, loadType));
+      }
+
+      return val;
+    }
+
+    // SQL Queries
+    boost::optional<double> ThermalZone_Impl::autosizedMaximumOutdoorAirFlowRate() const {
+      //TODO: I don't think this is available anywhere actually
+      return boost::none;
+    }
```

Comment:
* [ ] TODO: I can't find this one anywhere.
 
@asparke2 ideas please?

Reply:
@jmarrec I also don't see this used in openstudio-standards code or a value in the sql file I looked in, probably should remove the method.

Reply:
Removed in 981d0baa3062eaa1fcf10c700c6f04b81febd698